README.md

```diff
@@ -35,130 +31,70 @@ Welcome to the Netlify CLI! The new 2.0 version (now in beta) was rebuilt from t
 </details>
 <!-- AUTO-GENERATED-CONTENT:END -->
 
-## Installation
 
-**Prerequisites**
-
-- [Node.js](https://nodejs.org/en/download/) 8+.
-- [Netlify User Account](http://app.netlify.com/)
-
-To install the Netlify CLI, run the following command in your terminal:
-
-```sh-session
-npm install netlify-cli@next -g
-```
-
-## Usage
-
-After installing the CLI globally, connect the CLI to your Netlify account with the following command:
-
-```sh-session
-netlify login
-```
-
-This will open a browser window, asking you to log in with Netlify and grant access to **Netlify CLI**. This will store your Netlify access token in your home folder, under `~/.netlify/config.json`.
-
-```sh-session
-netlify [command]
-
-# Run `help` for detailed information about CLI commands
-netlify [command] help
-```
-
-### CI and Environment Variables.
-
-The following environment variables can be used to override configuration file lookups and prompts:
-
-- `NETLIFY_AUTH_TOKEN` - an access token to use when authenticating commands. **KEEP THIS VALUE PRIVATE**
-- `NETLIFY_SITE_ID` - override any linked site in the current working directory.
-
-## Getting Started + Docs
```

Comment:
We can remove this part though, since I already moved these links to the very tippy top of the README